### PR TITLE
Add fish shell tab completion for mac

### PR DESCRIPTION
**Pull request by chenrui333**
# Pull Request Check List

- [ ] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

-------

This PR just to add the documentation for fish shell tab completion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ poetry completions bash > $(brew --prefix)/etc/bash_completion.d/poetry.bash-com
 # Fish
 poetry completions fish > ~/.config/fish/completions/poetry.fish
 
+# Fish (macOS/Homebrew)
```
Comment (dawidd6) [inline README.md]: We have Homebrew on Linux too ;)

Comment (chenrui333) [reply README.md]: Is that the configuration the same?

Comment (dawidd6) [reply README.md]: Yep

Comment (chenrui333) [reply README.md]: Cool, I will update it.

**Comment by chenrui333** [comment]
cc @finswimmer

**Pull request merged**